frontend/src/Component/ApplicationForm.jsx

```
@@ -0,0 +1,111 @@
+            setLoading(false);
+        }
+    };
+
+
+    return (
+        <form onSubmit={handleSubmit} className="bg-gray-900 p-6 rounded-lg border border-gray-700 w-full text-white">
+            <h4 className="text-lg font-semibold mb-4">Fill in the application fields</h4>
+
+            <div className="grid grid-cols-1 md:grid-cols-2 gap-4 mb-4">
+                {placeholders.map((ph) => (<div key={ph} className="flex flex-col">
+                        <label className="mb-1 capitalize text-gray-300">{ph.replace(/_/g, ' ')}</label>
```

Biome (1.9.4)

[error] A form label must be associated with an input.

Consider adding a `for` or `htmlFor` attribute to the label element or moving the input element to inside the label element.

(lint/a11y/noLabelWithoutControl)